planet retrieve quad endpoint returns a different download URL

I noticed it in the last build: https://github.com/openforis/sepal_ui/actions/runs/11858277401/job/33048545863, test were failing because the response from the server has changed:

E       -  download: https://api.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/full?api_key=toto
E       +  download: https://link.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/full?api_key=toto


there's a difference, in the past it was `api.planet`, not it is `link.planet`... I have checked the changelog and their blog and I didn't find anything, both links work in any case, I will leave this issue open until I find what happens, or maybe it's a deprecationalert.

--- a/tests/test_planetapi/test_PlanetModel/test_get_quad.yml
+++ b/tests/test_planetapi/test_PlanetModel/test_get_quad.yml
@@ -1,12 +1,12 @@
 _links:
   _self: https://api.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058?api_key=toto
-  download: https://api.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/full?api_key=toto
+  download: https://link.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/full?api_key=toto
   items: https://api.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/items?api_key=toto
   thumbnail: https://tiles.planet.com/basemaps/v1/planet-tiles/planet_medres_normalized_analytic_2015-12_2016-05_mosaic/gmap/11/1088/989.png?api_key=toto
 bbox:
-- 11.2499999985
-- 5.9657536703
-- 11.4257812485
-- 6.14055478166
+  - 11.2499999985
+  - 5.9657536703
+  - 11.4257812485
+  - 6.14055478166
 id: 1088-1058
 percent_covered: 100

--- a/tests/test_planetapi/test_PlanetModel/test_get_quad.yml
+++ b/tests/test_planetapi/test_PlanetModel/test_get_quad.yml
@@ -4,9 +4,9 @@ _links:
   items: https://api.planet.com/basemaps/v1/mosaics/1f0570f3-b373-457f-89a5-046b4080d199/quads/1088-1058/items?api_key=toto
   thumbnail: https://tiles.planet.com/basemaps/v1/planet-tiles/planet_medres_normalized_analytic_2015-12_2016-05_mosaic/gmap/11/1088/989.png?api_key=toto
 bbox:
-  - 11.2499999985
-  - 5.9657536703
-  - 11.4257812485
-  - 6.14055478166
+- 11.2499999985
+- 5.9657536703
+- 11.4257812485
+- 6.14055478166
 id: 1088-1058
 percent_covered: 100

